node/node.go

```diff
@@ -174,6 +172,11 @@ func (n *Node) GetBalance(address string, options api.AccountQueryOptions) (*big
 func (n *Node) GetUsername(address string, options api.AccountQueryOptions) (string, api.BlockInfo, error) {
 	userAccount, blockInfo, err := n.loadUserAccountHandlerByAddress(address, options)
 	if err != nil {
+		adaptedBlockInfo, adaptErr := adaptBlockInfoIfError(err)
```

Comment:
`adaptErr` vs. `adaptedErr`.

If the suggestion below would be applied, then we can have `blockInfoOfEmptyAccount, isEmptyAccount := detectEmptyAccount(...)`.

Reply:
renamed / refactored